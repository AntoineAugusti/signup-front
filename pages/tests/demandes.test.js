import ReactTestRenderer from 'react-test-renderer'

describe('pages | demandes', () => {
  describe('render', () => {
    const renderer = ReactTestRenderer.create(<demandes />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
