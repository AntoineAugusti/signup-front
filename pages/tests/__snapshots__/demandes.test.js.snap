// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages | demandes render should match snapshot 1`] = `
<div
  className="page"
>
  <header>
    <nav
      className="nav"
    >
      <div
        className="nav__container"
      >
        <a
          className="nav__link"
          href="/"
        >
          <img
            alt="Accueil de particulier.api.data.gouv.fr"
            className="nav__logo"
            src="/static/images/logo-api-particulier.png"
          />
        </a>
        <ul
          className="nav__links"
        >
          <li>
            <a
              href="/about"
              onClick={[Function]}
            >
              A propos
            </a>
          </li>
          <li>
            <a
              href="/documentation"
              onClick={[Function]}
            >
              Documentation technique
            </a>
          </li>
          <li>
            <a
              href="/contractualisation"
              onClick={[Function]}
            >
              contractualisation
            </a>
          </li>
          <li>
            <a
              href="/contact"
              onClick={[Function]}
            >
              Contactez-nous
            </a>
          </li>
          <li>
            <a
              href="http://localhost:3022/oauth/authorize?client_id=82546188522214c3577d35c283ce8842786649b35a026a9d44908037a597f29b&redirect_uri=http%3A%2F%2Flocalhost%3A3000&scope=&response_type=token&state="
            >
              Se connecter
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </header>
  <section
    className="section-grey"
  >
    <div>
      <div
        className="container"
      >
        <h2
          className="section__title"
        >
          Vous devez vous connecter avant de continuer
        </h2>
      </div>
    </div>
    <div>
      <img
        alt={null}
        src={null}
      />
    </div>
  </section>
  <footer
    className="footer"
  >
    <div
      className="container"
    >
      <div
        className="footer__logo"
      >
        <a
          href="https://api.gouv.fr/"
        >
          <img
            alt="Logo Beta Gouv"
            src="/static/images/logo-api.gouv.fr-white.png"
          />
        </a>
        <ul
          className="footer__social"
        >
          <li>
            <a
              href="https://twitter.com/BetaGouv"
              title="Twitter"
            >
              <img
                alt="Twitter"
                className="icon icon-twitter"
                src="/static/images/social/twitter.svg"
              />
            </a>
          </li>
          <li>
            <a
              href="https://github.com/betagouv"
              title="Github"
            >
              <img
                alt="Github"
                className="icon icon-github"
                src="/static/images/social/github.svg"
              />
            </a>
          </li>
          <li>
            <a
              href="mailto:contact@particulier.api.gouv.fr"
              title="Nous écrire un mail"
            >
              <img
                alt="Email"
                className="icon icon-mail"
                src="/static/images/social/email.svg"
              />
            </a>
          </li>
        </ul>
      </div>
      <ul
        className="footer__links"
      >
        <li>
          <h2>
            particulier.api.gouv.fr
          </h2>
        </li>
        <li>
          <a
            href="https://www.data.gouv.fr/reference"
          >
            Données de référence
          </a>
        </li>
        <li>
          <a
            href="https://www.data.gouv.fr/terms"
          >
            Conditions générales d‘utilisation
          </a>
        </li>
        <li>
          <a
            href="https://www.data.gouv.fr/api"
          >
            API
          </a>
        </li>
      </ul>
    </div>
  </footer>
</div>
`;
